sdcm/cluster.py

```diff
@@ -270,10 +270,17 @@ def configure_rsyslog(cls, enable_ngrok=False):
     def get_startup_script(cls):
         post_boot_script = '#!/bin/bash'
         if cls.RSYSLOG_ADDRESS:
-            post_boot_script += dedent('''
-                   sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
-                   sudo systemctl restart rsyslog
-                   '''.format(*cls.RSYSLOG_ADDRESS))
+            if IP_SSH_CONNECTIONS == 'public' or Setup.MULTI_REGION:
+                post_boot_script += dedent('''
+                       sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
+                       sudo systemctl restart rsyslog
+                       '''.format('127.0.0.1', '5000'))
```

Comment:
create `SSH_TUNNEL_PORT` global in the `auto_ssh.py`

Reply:
Add a global for it